docs/workflow_tasks/queue_granules.md

```diff
@@ -0,0 +1,94 @@
+---
+id: queue_granules
```

Comment:
You need to add this page to the sidebar for `Workflow Tasks` for it to appear in `website/sidebars.json`